### ISSUE TITLE
No recovery images yet

### ISSUE THREAD

**Issue by razzintown** [MotorolaMobilityLLC/recovery-images#1]
Bug Overview: No recovery images in repo.

Steps to Reproduce: Clone repo, list files.

Actual Results: 
```
README.md
```

Expected Results: Recovery images should exist in repo (or code to build them?).

Additional Information: I see this repo is new, but there is no estimated timeline or any information in the readme. ;)

**Comment by tags07** [comment]
Sorry I was just testing.. Unfortunately github has a size limitation that prevents us from posting recovery images to github. We are going to continue to post recovery images on Motorola's website for now..till we figure a better option.

**Comment by razzintown** [comment]
I thought that recovery mode images were around 1-10MB. But you must mean entire system images.

Perhaps this is an option: https://help.github.com/articles/versioning-large-files/

**Comment by tags07** [comment]
Yes, we are posting system, recovery, fastboot tools, bootloader etc..
